examples/06-verif-manual/vm-006-pinched_cylinder.py

```diff
@@ -0,0 +1,425 @@
+   H. Takemoto, R. D. Cook, "Some Modifications of an Isoparametric Shell
+   Element", International Journal for Numerical Methods in Engineering, Vol.7
+   No. 3, 1973.
+
+Analysis Type(s):
+ - Static Analysis ``ANTYPE=0``
+
+Element Type(s):
+ - 4-Node Finite Strain Shell Elements (SHELL181)
+ - 8-Node Finite Strain Shell Elements (SHELL281)
+
+.. image:: ../../_static/vm6_setup.png
+   :width: 400
+   :alt: VM6 Pinched Cylinder Problem Sketch
+
+Material Properties
+ - :math:`E = 10.5 \cdot 10^6 psi`
+ - :math:`Nu = 0.3125`
+
+Geometric Properties:
+ - :math:`l = 10.35  in`
+ - :math:`r = 4.953  in`
+ - :math:`t = 0.094  in`
+
+Loading:
+ - :math:`F = 100  lb`
+
+Analysis Assumptions and Modeling Notes:
+ - A one-eighth symmetry model is used. One-fourth of the load is applied
+   due to symmetry.
+
+"""
+# sphinx_gallery_thumbnail_path = '_static/vm6_setup.png'
+
+###############################################################################
+# Start MAPDL
+# ~~~~~~~~~~~
+
+from ansys.mapdl.core import launch_mapdl
+
+# Start mapdl.
+mapdl = launch_mapdl()
+
+
+###############################################################################
+# Initiate Pre-Processing
+# ~~~~~~~~~~~~~~~~~~~~~~~
+# Enter verification example mode and the pre-processing routine.
+
+def start_prep7():
+    mapdl.clear()
+    mapdl.verify()
+    mapdl.prep7()
+
+# Call the function.
+start_prep7()
+
+
+###############################################################################
+# Define Element Type
+# ~~~~~~~~~~~~~~~~~~~
+# Set up the element type (a shell-type).
+
+# Define the element type number.
+def define_element(elem_type):
+    # Type of analysis: Static.
+    mapdl.antype("STATIC")
+
+    # Define the element type number.
+    elem_num = 1
+
+    if elem_type == "SHELL181":
+
+        # Element type: SHELL181.
+        mapdl.et(elem_num, elem_type)
+
+        # Special Features are defined by keyoptions of shell element:
+
+        # KEYOPT(3)
+        # Integration option:
+        # Full integration with incompatible modes.
+        mapdl.keyopt(elem_num, 3, 2)  # Cubic shape function
+
+    elif elem_type == "SHELL281":
+
+        # Element type: SHELL181.
+        mapdl.et(elem_num, "SHELL281")
+
+    # Show currently defined element type.
+    print(mapdl.etlist())
+    return elem_type
+
+# Call the function and get the number of the element type.
+elem_type = define_element(elem_type="SHELL181")
+
+
+###############################################################################
+# Define Material
+# ~~~~~~~~~~~~~~~
+# Set up the material properties, where:
+# Young Modulus is :math:`E = 10.5 \cdot 10^6 psi`,
+# Poisson's ratio is :math:`Nu = 0.3125`.
+
+# Define material number.
+mat_num = 1
+
+# Define material properties.
+def define_material():
+    # Define material properties.
+    mapdl.mp("EX", mat_num, 10.5E6)
+    mapdl.mp("NUXY", mat_num, 0.3125)
+    print(mapdl.mplist())
+
+# Call the function.
+define_material()
```

Comment:
Apply this to the rest of the cell blocks. Print statements should not be within the functions.

```suggestion
def define_material():
    # Define material properties.
    mapdl.mp("EX", mat_num, 10.5E6)
    mapdl.mp("NUXY", mat_num, 0.3125)
    return mapdl.mplist()

resp = define_material()
print(resp)
```

Reply:
Seems that this is outdated. Please move `print` outside of `define_material`. Printing should be avoided within methods, let's handle the return outside the function.